### PR TITLE
Install evernote, gvm and oh-my-zsh

### PR DESCRIPTION
**Pull request by RaviH**

### CODE REVIEW

File: zsh.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+- hosts: workstation
+
+  tasks:
+    - name: get zsh script
+      get_url: url=http://github.com/robbyrussell/oh-my-zsh/raw/master/tools/install.sh dest=/tmp/zsh.sh
+
+    - name: change file permission to be executable
+      command: chmod +x /tmp/zsh.sh
+
+    - name: install zsh
```
Comment (spencergibb) [inline zsh.yml]: it's not installing zsh, it's installing `oh-my-zsh`, please change the names here.

Comment (spencergibb) [inline zsh.yml]: and the name of the yml file

File: cask.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+- hosts: workstation
+
+  tasks:
+    - name: install mysqlworkbench
```
Comment (spencergibb) [inline cask.yml]: wrong name

**Comment by RaviH** [comment]
Fixed as per PR comment!

**Pull request merged**